internal/api/mod.go

```diff
@@ -60,7 +63,13 @@ func (m Mod) Download(ctx *cli.Context) error {
 	}
 
 	for _, dependency := range cfg.Deps {
-		if err := cmd.Get(ctx.Context, dependency); err != nil {
+		err := cmd.Get(ctx.Context, dependency)
+		if err != nil {
+			if errors.Is(err, models.ErrVersionNotFound) {
+				slog.Warn("Version not found", "dependency", dependency)
```

Comment:
this is definitely a error, not warn

Moreover, it looks as if the result will be code 0, since the error is skipped

Reply:
yeap, i'll fix it later